# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpChannelChild.h
# Commit: 673030046319
# Full Hash: 673030046319948020e1e2100e2db3cf753dbaba
# Author: Junior Hsu <juhsu@mozilla.com>
# Date: 2020-06-30 10:00:42
# Regressor Bug: 1633935
# File Overlap Count: 1
# Description:
#   Bug 1633935 - P9 send OnAfterLastPart via pBg IPC to avoid race between OS*R, r=mayhemer,necko-reviewers
#   
#   Depends on D77750
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77751
# ==============================================================================

diff -r 045271cd2ca6 -r 673030046319 netwerk/protocol/http/HttpChannelChild.h
--- a/netwerk/protocol/http/HttpChannelChild.h	Tue Jun 30 00:33:41 2020 +0000
+++ b/netwerk/protocol/http/HttpChannelChild.h	Tue Jun 30 00:33:54 2020 +0000
@@ -176,8 +176,6 @@
   mozilla::ipc::IPCResult RecvSetClassifierMatchedTrackingInfo(
       const ClassifierInfo& info) override;
 
-  mozilla::ipc::IPCResult RecvOnAfterLastPart(const nsresult& aStatus) override;
-
   virtual void ActorDestroy(ActorDestroyReason aWhy) override;
 
   virtual void DoNotifyListenerCleanup() override;
@@ -266,6 +264,7 @@
       const nsTArray<ConsoleReportCollected>& aConsoleReports);
   void ProcessFlushedForDiversion();
   void ProcessDivertMessages();
+  void ProcessOnAfterLastPart(const nsresult& aStatus);
   void ProcessOnProgress(const int64_t& aProgress, const int64_t& aProgressMax);
 
   void ProcessOnStatus(const nsresult& aStatus);